tests/cli/upload/conftest.py

```diff
@@ -163,10 +241,33 @@ def files(self, **kwargs):
             return MockFile(empty_first=True)
         return MockFile()
 
+    def get_files(self, bundle, tags, version="1.0"):
+        """docstring for get_files"""
```

Comment:
```suggestion
        """Mock get files sub from Housekeeper"""
```
I know it is only a test function, but I do not think it is much use to have docstring with so little context description. It would help with some more info 😄